### DISCUSSION TITLE
Help with imported cmake project errors;

### DISCUSSION THREAD

**Discussion by oDeenBrowno** [eclipse-cdt/cdt#466] [Q&A]
Hi, I am practicing importing C++ projects that have makefiles, cmakelists, etc. I import the projects and get index errors, build errors, and include errors.

Is there anyone who might be able to assist me with where I am going wrong?

**Comment by jonahgraham** [comment]
We can try to help - please provide specific examples and we'll see what we can do.

**Reply by oDeenBrowno** [reply]
[youtube_video](https://www.youtube.com/watch?v=krTKfa0OYAc)

Hi I am a novice user of Eclipse, I have not used the program before I signed up for an online course to program the raspberry pi pico. In the video I am opening a project folder containing files created by the instructor. In the online course the files can be opened in Eclipse without any error.

I however, get indexing errors as shown in the video.

I have attached a zip file with some the folders I am trying to open, I am curious if it is my machine that is the problem or some other issue.

[JDBlink1.zip](https://github.com/eclipse-cdt/cdt/files/12140155/JDBlink1.zip)

**Reply by oDeenBrowno** [reply]
Any feedback?

**Reply by jonahgraham** [reply]
Sorry - nothing much to offer here. The instructor should probably be the first step, but I do see at least one bug, the null pointer exception:

![image](https://github.com/eclipse-cdt/cdt/assets/679236/14862172-2c3b-4b5c-a565-b0ae06ef5142)

If you can provide the full exception from the log we may be able to fix that.

---

In general it looks like these projects are mixing and matching multiple workflows. The projects have CMake setup, but also using Managed Build, including the Embed CDT workflows for tool chain management.

I don't know what your instructor had hope for here, but the Embed CDT project has a nice tutorial on creating a blinky project [here](https://eclipse-embed-cdt.github.io/tutorials/blinky-arm/)
